ux.symfony.com/config/packages/pwa.yaml

```diff
@@ -0,0 +1,50 @@
+pwa:
+    manifest_filepath: "%kernel.project_dir%/public/site.webmanifest"
+    image_processor: 'pwa.image_processor.imagick'
+    background_color: "#ffffff"
+    theme_color: "#212529"
+    name: 'Symfony UX'
+    short_name: 'symfony-ux'
+    description: 'A set of PHP & JavaScript packages, curated by Symfony, to solve your every day frontend problems. Featuring Stimulus and Turbo.'
+    orientation: "any"
+    display: "standalone"
+    scope: "/"
+    display_override: ['fullscreen', 'minimal-ui', 'window-controls-overlay']
+    id: "/"
+    start_url: "./"
+    icons:
+        - src: "%kernel.project_dir%/assets/images/pwa/logo.svg"
+          format: 'png'
+          sizes: [48, 57, 60, 72, 76, 96, 114, 128, 144, 152, 180, 192, 256, 384, 512, 1024]
+        - src: "%kernel.project_dir%/assets/images/pwa/logo.svg"
+          purpose: 'maskable'
+          format: 'png'
+          sizes: [48, 57, 60, 72, 76, 96, 114, 128, 144, 152, 180, 192, 256, 384, 512, 1024]
+    screenshots:
+        - src: "%kernel.project_dir%/assets/images/pwa/screenshots/desktop/"
+          form_factor: "wide"
+
+        - src: "%kernel.project_dir%/assets/images/pwa/screenshots/mobile/"
+          form_factor: "narrow"
+    categories: ['utility', 'productivity']
+    shortcuts:
+        - name: "Turbo"
+          short_name: "turbo"
+          description: "See Turbo in live action"
+          url: "app_turbo"
+          icons:
+              - src: "%kernel.project_dir%/assets/images/pwa/turbo.svg"
+                format: 'png'
+                sizes: [48, 96, 256]
+        - name: "Live Components"
+          short_name: "live-components"
+          description: "An Interactive UI entirely built in Twig"
+          url: "app_live_component"
+          icons:
+              - src: "%kernel.project_dir%/assets/images/pwa/component.svg"
+                format: 'png'
+                sizes: [48, 96, 256]
+    serviceworker:
+        generate: true
+        filepath: "%kernel.project_dir%/public/sw.js"
+        src: "/sw.js"
```

Comment:
Does the bundle generate this file (I see it committed in this PR - but was it generated)?

In a real app, would you want this file - like other .js files - to have a version hash in the filename to bust browser cache on updates?

Reply:
I think I will split the serviceworker and manifest generation.
The serviceworker is supposed to be created once and modified depending on the application needs.
The manifest could be re-generated as the application evolves.

Reply:
OK it is now a dedicated command `pwa:sw`